### PR TITLE
Use planemask constants where applicable

### PR DESCRIPTION
**Pull request by tgsm**

### CODE REVIEW

File: include/gx_layers.h [hunk 1]
```diff
@@ -48,10 +48,10 @@ typedef struct GraphicsBanks {
 
 void GX_SetBanks(const GraphicsBanks *banks);
 void GX_DisableEngineALayers(void);
-void GX_EngineAToggleLayers(u32 layer_mask, GXLayerToggle enable);
+void GX_EngineAToggleLayers(GXPlaneMask planeMask, GXLayerToggle enable);
```
Comment (adrienntindall) [inline include/gx_layers.h:51]: Should we change the names of these functions now that we know theyre plane mask toggles

Comment (red031000) [reply include/gx_layers.h:51]: I already made an issue of this, yes we really should

**Comment by red031000** [comment]
wondering if it'd be better to use GfGfx, cause we use that elsewhere, or change the GfGfx to GF_GX_

**Review by red031000 — APPROVED** [review]

**Comment by red031000** [comment]
build failed

**Comment by tgsm** [comment]
Merging based on prior approval

**Pull request merged**